Formula/glibc.rb

```diff
@@ -195,4 +203,13 @@ def post_install
     safe_system "#{lib}/libc.so.6", "--version"
     safe_system "#{bin}/locale", "--version"
   end
+
+  def ld_so_conf
+    <<~EOS
+      include #{etc}/ld.so.conf.d/*.conf
+      include #{opt_prefix}/etc/ld.so.conf.d/*.conf
```

Comment:
Can we add `#{lib}` right above `HOMEBREW_PREFIX/lib`?   That will protect against `brew unlink` breaking things, because it means `glibc` will always at least be able to find its own libraries even if it is unlinked from `HOMEBREW_PREFIX/lib`.